Ginger/GingerCoreNET/Logger/WebReportGenerator.cs

```diff
@@ -129,68 +130,93 @@ public void DeleteFoldersData(string clientAppFolderPath)
 
         private void PopulateMissingFields(LiteDbRunSet liteDbRunSet, string clientAppPath)
         {
+            //select template 
+            HTMLReportConfiguration _HTMLReportConfig = WorkSpace.Instance.SolutionRepository.GetAllRepositoryItems<HTMLReportConfiguration>().Where(x => (x.IsDefault == true)).FirstOrDefault();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4549121)